var.c

```diff
@@ -52,14 +52,21 @@ extern bool varassign_string(char *extdef) {
 	return TRUE;
 }
 
+char* get_message(char *format, char *msg) {
```

Comment:
Prefer `char *`-style decl syntax

Reply:
This whole helper function actually is unneeded -- and allocating error messages with malloc will leak memory.

Incidentally, `trip.rc` probably doesn't verify leak-free behavior but at one time I ran rc through valgrind to catch leaks. I wonder if the state of the art of leak detection for C has advanced to the point where we could run a tool in-line for pull requests.

Reply:
It's a while since I ran this, but AFAICR trip.rc only succeeds with this flag OFF. Because there are undefined variables being used in there 😃 .  Notwithstanding I have been using this branch as my daily drive. I will have a look at this again.

Reply:
trip.rc can exercise features selectively. It's not all-or-nothing.